source/CommonJobs/CommonJobs.MVC.UI/Scripts/Evaluations/calification.js

```diff
@@ -68,6 +64,20 @@
             return  calification.evaluatorEmployee == self.userLogged;
         }).comments;
 
+        if (this.userView == 3) {
```

Comment:
I'd suggest avoiding magic numbers. At the very least we can setup "constants" in JS and use those.